README.md

```diff
@@ -1 +1,58 @@
-# MediaMonks - Frontend Coding Standards
+# frontend-coding-standards
+
+## Recommended Libraries
+
+#### Animation & UI
+ * [classnames](https://www.npmjs.com/package/classnames) - A simple JavaScript utility for conditionally joining classNames together.
+ * [Draggable](https://greensock.com/draggable/) - Green Sock library for dragging elements
+ * [grid-checker](https://www.npmjs.com/package/grid-checker) - provides developers a customizable, responsive grid-overlay to compare with the layout provided by design to compare layouts
+ * [GSAP](https://greensock.com/gsap/) - Green Sock Animation Platform -Animation library
+ * [Litepicker](https://github.com/ThijsTyZ/Litepicker) - Date range picker - lightweight, no dependencies
+ * [Lottie](http://airbnb.io/lottie/) - parses Adobe After Effects animations exported as json with Bodymovin and renders them natively on mobile and on the web
+ * [scroll-tracker-component-manager](https://www.npmjs.com/package/scroll-tracker-component-manager) - The ScrollTrackerComponentManager is a Class that tracks whether a component is within your viewport based on your scroll position.
+ * [seng-scss](https://www.npmjs.com/package/seng-scss) - A SCSS library of mixins and functions
+ * [seng-device-state-tracker](https://github.com/mediamonks/seng-device-state-tracker) - DeviceStateTracker is a utility class that tracks which media query is currently active using the matchMedia API.
+ * [storybook](https://github.com/storybookjs/storybook) - UI component dev & test
+
+#### Coding
+ * [axios](https://github.com/axios/axios) - Promise based HTTP client
+ * [bowser](https://www.npmjs.com/package/bowser) - A small, fast and rich-API browser/platform/engine detector
+ * [date-fns](https://www.npmjs.com/package/date-fns) - provides the most comprehensive, yet simple and consistent toolset for manipulating JavaScript dates
+ * [i18next](https://www.npmjs.com/package/i18next) - Internationalization library
+ * [lodash](https://lodash.com/) - A modern JavaScript utility library delivering modularity, performance & extras.
+ * [Moment](https://www.npmjs.com/package/moment) - A lightweight JavaScript date library for parsing, validating, manipulating, and formatting dates.
```

Comment:
I would not ever recommend moment because of mutability and importing the whole moment object. Date-fns solves date handling in a much cleaner way. The codebase then deals with native Date objects, not Moment objects and you only import the functions you need. Manipulation is very clear because you always get a new Date. With moment you need to remember to clone the object...

Reply:
I agree. However I see that this library is still used a lot on projects. So should we explicit discourage the usage of this library?

Reply:
At least the description should change, it’s not lightweight and it has a lot of (forementioned) caveates, perhaps it should be on the 'libraries to consider' list but not be a recommendation

Reply:
While I agree with most of what is said, I still heavily favour `moment` when working with **timezones**.

Using timezones with `date-fns`, that _changes the actual hours_ and ignores the timezone when "changing timezones", feels super flaky.

So in general I prefer date-fns, but for timezones I prefer moment.

Reply:
How about [date-fns-tz](https://github.com/marnusw/date-fns-tz) to complement date-fns? I have no personal experience with it though but it appears to solve this particular need.

Reply:
We previously used [timezone-support](https://www.npmjs.com/package/timezone-support), which seems to be used by `date-fns-tz` as well, so the experience should be slightly improved.

Still, the issue is that you cannot have "a date in a timezone" as an object using date-fns, because everything is passed along as a internal date, which **always** has the local timezone.

Which is kind of okay during formatting (just pass the timezone when formatting), but gets annoying when having to use getters for different date units; you have to pass the timezone to every function you use (seems that getting hours of a date in a timezone as a number is not even supported in those libs).

So it's possible, but takes a lot more effort, with a higher chance of errors (in my opinion).

Reply:
> Considering using Moment in your project?
> There may be better modern alternatives.
https://momentjs.com/

Event Moment.js itself is not recommending it anymore, so I will remove it